src/config.js

```diff
@@ -31,6 +31,8 @@ var config;
 
 // If the developer hasn't put anything in local.js then give them
 // the values from staging.
+
```

Comment:
@AnkitaKhurana Can you remove these newlines added to `src/config.js` as part of this commit.